source/includes/connection-snippets/scram-snippets/php-connection.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/* Begin scram connection */
+$client = new MongoDB\Client(
+    'mongodb+srv://<username>:<password>@<cluster-url>/test?retryWrites=true&w=majority'
+);
+
+$collection = $client->testDB->testCol;
+$docCount = $collection->count([]);
+print($docCount);
```

Comment:
The [`echo`](https://www.php.net/echo) language construct is more idiomatic for PHP. I'd suggest:

```php
echo $docCount, "\n";
```